website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -84,52 +66,48 @@ The Vector [Aggregator] lets you [transform] and ship data collected by other ag
 To check available Helm chart configuration options:
 
 ```shell
-helm show values vector/vector-aggregator
+helm show values vector/vector
 ```
 
-This example configuration file lets you use Vector as an Aggregator to parse events to make them human-readable. For more information about configuration options, see the [Configuration] docs page.
+An Aggregator by default is configured to accept events from a variety of common sources and writes them to standard output, it also reports internal metrics via a Prometheus exporter. For more information about configuration options, see the [Configuration] docs page.
+
+The default sources and their associated ports are listed below:
```

Comment:
We may want to add a comment in the helm repo to update this whenever the defaults change.

Reply:
🤔 good point, maybe it would be better to direct users to the config file/a command to show what it is rather than list here - to ensure they stay in sync?

Reply:
That'd be fine too 👍

Reply:
Struggling to think of a way to do that actually. Do we just want to link to https://github.com/vectordotdev/helm-charts/blob/develop/charts/vector/templates/configmap.yaml ?

Reply:
Yeah, I think that would be fine. It also helps raise awareness of the repo.

Reply:
👍🏻 on linking these. Less things to sync up, and I'm not sure what benefit listing these here gives.

Reply:
I mostly didn't want it to be: "we start a variety of things by default, figure out what and how to reach them on your own :middle_finger_emoji:"

Reply:
😄 that is where I think the link would help.

Reply:
Just a reminder to update this to a link.